src/evilHangman/hangmanLetterButtons.ts

```diff
@@ -0,0 +1,17 @@
+import { hangmanLetterButton, Letter } from '../buttons/hangmanLetterButton';
+
+const alphabet: Array<Letter> = new Array(26)
+	.fill(null)
+	.map((x, i) => String.fromCharCode(i + 97) as Letter);
+
+const buttonMap: Map<Letter, Button> = new Map(
+	alphabet.map(letter => [letter, hangmanLetterButton(letter)])
+);
+
+export const letterButtons: Array<Button> = [...buttonMap.values()];
+
+export function getLetterOptions(guessesSoFar: Set<string>): Array<Button> {
```

Comment:
Hard to tell from looking at call sites that this function filters the contents of the given set from our alphabet. Can we rename this to something more descriptive, perhaps with a JSDoc comment?

Reply:
done